pycoast/cw_base.py

```diff
@@ -1034,27 +1062,32 @@ def add_points(self, image, area_def, points_list, font_file, font_size=12,
             points_list : list [((lon, lat), desc)]
               | a list of points defined with (lon, lat) in float and a desc in string
               | [((lon1, lat1), desc1), ((lon2, lat2), desc2)]
+              | See coord_ref (below) for meaning of lon,lat
               | lon : float
-              |    longitude of a point
+              |    longitude or pixel x of a point
               | lat : float
-              |    latitude of a point
+              |    latitude or pixel y of a point
```

Comment:
Since the `(lon, lat)` in the doc string refers to `(longitude, latitude)` or `(x, y)` in this change; I was wondering should it be changed to some thing else like `(coord1, coord2)` to avoid misunderstanding.